### PR TITLE
LP-2316 Update course detail for mini-lesson

### PR DESCRIPTION
**Pull request by harris-irfan**
### Description

[LP-2316](https://philanthropyu.atlassian.net/browse/LP-2316)

[philu-edx-theme PR](https://github.com/philanthropy-u/philu-edx-theme/pull/###)

### Notes

- iSort ran on both files
- Unit tests for course_about view will be written in a separate story which is already created and ready for dev
- Linting for views.py file will also be done in a separate linting ticket of philu_overrides, which is also ready for dev
- Unit test related to the change in opendedx/features/cms/helpers.initialize_course_settings is not incorporated, because the pre-written unit test of the initialize_course_settings method is broken. Fixing that unit test would require extra, out-of-scope effort, and hence a separate story will be created for that.
- Linting for the entire cms app is also to be done in a separate linting story which is read for dev.
- Hence, for this PR, there are no unit tests, or no linting changes/report.

### CODE REVIEW

**Comment by philanthropyu** [comment]
Your PR has finished running tests. The following contexts failed:
* jenkins/python

**Review by ahmed-faraz46 — APPROVED** [review]
LGTM 👍 Just an optional suggestion.
Great refactoring and addition in `initialize_course_settings` method 👏

File: lms/djangoapps/philu_overrides/views.py [hunk 1]
```diff
@@ -613,6 +615,8 @@ def course_about(request, course_id):
 
         social_sharing_urls = get_social_sharing_urls(url_query_cleaner(request.build_absolute_uri()), meta_tags)
 
+        is_mini_lesson = custom_settings.is_mini_lesson
```
[the remark this answers is not in the document]
Comment (harris-irfan) [reply lms/djangoapps/philu_overrides/views.py:618]: I feel it's better the way it is, keeping in mind conceptual affinity and vertical distance. Meta tags are used right after they are fetched. if `is_mini_lesson` if fetched there, there would be a lot of vertical distance between its initialization and usage.

**Review by muak50 — APPROVED** [review]
LGTM 👍

**Pull request merged**